tests/component-tests/uiOptions/js/UIOptionsTests.js

```diff
@@ -220,6 +220,38 @@ https://github.com/fluid-project/infusion/raw/main/Infusion-LICENSE.txt
         }
     });
 
+    fluid.tests.uiOptions.multilingual = function (locale, direction) {
+        jqUnit.test("Test Multilingual UIO with locale " + locale + " direction " + direction, function () {
+            jqUnit.expect(2);
+            var togo = fluid.promise();
+            var check = function (that) {
+                var prefsEditor = that.prefsEditorLoader.prefsEditor;
+                jqUnit.assertEquals("Multilingual UIO loaded with correct container locale", locale, prefsEditor.container.attr("lang"));
+                jqUnit.assertEquals("Multilingual UIO loaded with correct container direction", direction, prefsEditor.container.attr("dir"));
+                that.destroy();
+                togo.resolve();
+            };
+            fluid.uiOptions.multilingual(".flc-prefsEditor-separatedPanel", {
+                gradeNames: "fluid.tests.uiOptions.testPrefsEditorBase",
+                locale: locale,
+                direction: direction,
+                listeners: {
+                    "onReady.check": {
+                        func: check,
+                        args: "{that}"
+                    }
+                }
+            });
+            return togo;
+        });
+    };
+
+    fluid.tests.uiOptions.multilingual("pt_BR", "", true);
+    fluid.tests.uiOptions.multilingual("pt_BR", "rtl", true);
```

Comment:
You might want to use the `fa` (Farsi) message bundle which is a right-to-left language. Probably doesn't matter for the test, but might be clearer for someone reading it in the future.